### PR TITLE
feat: create pull request on the default branch

### LINKED ISSUE #4: Support default branches other than master
**Is your feature request related to a problem? Please describe.**
Currently only supports pull-requests again master

**Describe the solution you'd like**
Should look at default branch, and base pull-reuqest against that branch

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sinchang**
closes #4

### CODE REVIEW

File: src/Repository/index.js [hunk 1]
```diff
@@ -82,6 +82,15 @@ class Repository {
         })
     }
 
+    async getDefaultBranch() {
+        const result = await this.github.repos.get({
```
Comment (jakebolam) [inline src/Repository/index.js]: I'm wary of introducing more network requests. It looks like we can get this from the initial payload in: https://github.com/all-contributors/all-contributors-bot/blob/master/test/fixtures/issue_comment.created.json#L181

Comment (jakebolam) [reply src/Repository/index.js]: (we want to keep the number of API requests down, I feel like it's already pretty high)

File: src/Repository/index.js [hunk 2]
```diff
@@ -61,17 +61,17 @@ class Repository {
...
             repo: this.repo,
-            ref: `heads/master`,
+            ref: `heads/${defaultBranch}`,
         })
         return result.data.object.sha
     }
 
-    async createBranch(branchName) {
-        const fromSha = await this.getHeadRef()
+    async createBranch(branchName, defaultBranch) {
```
Comment (jakebolam) [inline src/Repository/index.js]: :tada:

Comment (jakebolam) [reply src/Repository/index.js]: [minor] usually switch parameters to named, if more than 1 parameter to the function

**Comment by sinchang** [comment]
@jakebolam ready to merge. please review again.

**Comment by jakebolam** [comment]
Thanks @sinchang looks great! 🎉

**Pull request merged**